### DISCUSSION TITLE
"Open in ComfyUI" button to open the file's workflow directly in ComfyUI

### DISCUSSION THREAD

**Discussion by lilixil** [receyuki/stable-diffusion-prompt-reader#43] [Ideas]
With the [gallery view update](https://github.com/receyuki/stable-diffusion-prompt-reader/discussions/31), stable-diffusion-prompt-reader could potentially become an even more valuable tool. Just imagine if it had an 'Open in ComfyUI' button that would allow you to directly open the generated graphic, complete with all nodes and their workflow, directly in ComfyUI. With this feature, there would be no need to manually drag and drop files, eliminating the need to switch between folders.

I'm not sure if this is realistically feasible, but it would be a fantastic addition, especially considering that the program is lightweight and not resource-intensive.

**Comment by receyuki** [comment]
This is possible, but I need to complete the user preference panel first so that there is a place to enter your ComfyUI server address lol.

**Comment by lilixil** [comment]
Awesome!
